### ISSUE TITLE
[Feature Request] Keypad tones

### ISSUE THREAD

**Issue by prg318** [HenriDellal/emerald-dialer#14]
It would be great if there were an option for the keypad to produce telephone-style keypad tones when each key on the keypad is pressed, old telephone and like the AOSP dialer.

Great work on this program by the way - I wanted to pass this feature request along.

**Comment by HenriDellal** [comment]
Thanks for the feedback. It shouldn't be hard to do. Will do something about it later.

**Comment by vertigo220** [comment]
Agree, though it should be optional with the ability to adjust the volume of the tones. Also, vibrate on keypress would be nice and is related to this.
